client/web/compose/src/mixins/record.js

```diff
@@ -318,6 +318,10 @@ export default {
           this.fields = []
           this.record = new compose.Record(this.module, {})
           this.$emit('save')
+
+          this.$root.$emit('trigger-recordlist-refresh', {
```

Comment:
this will have to be called something different since its not a refresh but a record update that triggers this.
So I would say you should call it something along the lines of `module-records-updated`.
You should also just emit the moduleID, if you don't need anything else.